<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrugEditView</class>
 <widget class="QWidget" name="DrugEditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>药品编辑</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    background-color: #4CAF50; /* Green */
    color: white;
    padding: 10px 20px;
    text-align: center;
    text-decoration: none;
    display: inline-block;
    font-size: 16px;
    margin: 4px 2px;
    cursor: pointer;
    border: none;
    border-radius: 8px;
}

QPushButton:hover {
    background-color: #45a049;
}

QSpinBox, QComboBox, QLineEdit {
    padding: 5px;
    border: 2px solid #ccc;
    border-radius: 4px;
    width: 200px;
}

QLabel {
    font-size: 14px;
    padding: 5px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_id">
       <property name="text">
        <string>药品编号:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="dbEditID"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_name">
       <property name="text">
        <string>药品名称:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="dbEditName"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_stock">
       <property name="text">
        <string>库存数量:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="dbEditInventory"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_type">
       <property name="text">
        <string>药品类型:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="dbComboType">
       <item>
        <property name="text">
         <string>处方</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>非处方</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_dosage">
       <property name="text">
        <string>药品剂量:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="dbEditDose"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="btCancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btSave">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
